Gems/Atom/RHI/DX12/Code/Source/RHI/FrameGraphExecuteGroup.cpp

```diff
@@ -48,6 +48,7 @@ namespace AZ
 
             InitRequest request;
             request.m_scopeId = scope.GetId();
+            request.m_submitCount = scope.GetEstimatedItemCount();
```

Comment:
you will need to update the metal version as well. It should be similar to dx12

Reply:
Updated.